.changeset/fast-dodos-compete.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/admin": minor
+---
+
+Add `ForcePromptRoute`, a `Route` that triggers Prompt even if it is a subroute
+
+Use it in `StackSwitch`, so that navigating to a nested stack subpage will show a Prompt (if dirty)
```

Comment:
```suggestion
Used in `StackSwitch` so that navigating to a nested stack subpage will show a prompt (if dirty)
```

It's not necessary to change anything in the application, right?

Reply:
yes, when StackSwitch is used no changes are needed.